python/tests/test_cli/test_add.py

```diff
@@ -28,9 +28,11 @@ def test_add_github(self) -> None:
             ),
         )
 
-    def test_add_serpapi(self) -> None:
-        """Test `composio add` with no-auth."""
-        self.run("add", "serpapi", input="y")
+    def test_add_auth_mode_auto_uppercase(self) -> None:
```

Comment:
Consider keeping the `test_add_serpapi` test or adding it back with a different name, as it tests a different functionality (API key auth scenario) which is still valuable for test coverage.